python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -40,17 +40,22 @@ def on_forward_start(self, inputs):
         self._model_exists_or_err()
         return inputs
 
-    def forward_step(self, *inputs):
+    def forward_step(self, *inputs, **kwargs):
         flattened_inputs = []
         _flatten(inputs, flattened_inputs)
+        args_length = len(inputs)
+        if kwargs is not None and len(kwargs) > 0:
+            for arg in self.forward_args[args_length:]:
```

Comment:
maybe the `kwargs` contains the same argument as the `inputs`, I mean if the signature is `def forward(a, b, c)`, then user call `model(t1, t2, b=t3, c=t4)`, I think we should give an error in such case, not just ignore the `b=t3`.

Reply:
Good catch, thanks! Have fixed this.